server/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const path = require('path');
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, '/../public/')));
+
+app.get('/', (req, res) => {
+  res.end();
+})
```

Comment:
Not sure if this is necessary. I think you could remove it and everything would still work correctly.